jazz_create-serverless-service/components/crud/update.js

```diff
@@ -0,0 +1,53 @@
+const request = require('request');
+
+module.exports = (service_id, inputs, callback) => {
+    	
+	var jsonBody = getServiceData(inputs.body);	
+	var svcPayload = {
+		uri: inputs.SERVICE_API_URL + inputs.SERVICE_API_RESOURCE + "/" + service_id,
+		method: 'PUT',
+		headers: {'Authorization': inputs.TOKEN },
+		json: jsonBody,
+		rejectUnauthorized: false
+	};	
+	
+	request(svcPayload, function (error, response, body) {
+		if (response.statusCode === 200 && typeof body !== undefined && typeof body.data !== undefined) {
```

Comment:
Can we use the short form `body && body.data`

Reply:
Changed as suggested.